go.mod

```
@@ -57,8 +61,8 @@ require (
 	// repo is the first obvious idea, but has to be carefully considered, automated, and is not
 	// a hard blocker.
 	github.com/pokt-network/shannon-sdk v0.0.0-20240814144717-dfa95b525d46
+	// TODO_IB_THIS_PR: bump after https://github.com/pokt-network/smt/pull/52 is in
```

Not sure if it's intentional but this isn't being picked up by CI because of a typo:

```suggestion
	// TODO_IN_THIS_PR: bump after https://github.com/pokt-network/smt/pull/52 is in
```